### PR TITLE
add partner name for create, update and show commands for managementp…

### PR DESCRIPTION
**Pull request by jeffrey-ACE**
…artner

---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [x] Have you run `./scripts/ci/test_static.sh` locally? (`pip install pylint flake8` required)
- [x] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [x] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli-extensions/blob/master/docs/extension_summary_guidelines.md).

### CODE REVIEW

**Comment by derekbekoe** [comment]
Please change the description in setup.py to conform to https://github.com/Azure/azure-cli-extensions/blob/master/docs/extension_summary_guidelines.md
e.g. "Support for Management Partner preview." would be sufficient.

**Comment by jeffrey-ACE** [comment]
@derekbekoe Fixed the comment, can you please approve?

File: src/managementpartner/azext_managementpartner/managementpartner/version.py [hunk 1]
```diff
@@ -9,4 +9,5 @@
 # regenerated.
 # --------------------------------------------------------------------------
 
-VERSION = "0.1.0"
+VERSION = "0.1.1"
```
Comment (derekbekoe) [inline src/managementpartner/azext_managementpartner/managementpartner/version.py]: This version seems incorrect or is it different to the version in setup.py?

Comment (jeffrey-ACE) [reply src/managementpartner/azext_managementpartner/managementpartner/version.py]: Good catch, let me fix it

**Pull request merged**